Topology/README.md

```diff
@@ -35,3 +35,39 @@ Other possible quantities:
   - label (list of str) - Per-atom labels which may be seperate from fragments
   - Extend the `real` quantitity to cover real, ghost, absent, qm/mm region, etc.
   - EFP quantities `fragment_types`, `coordinate_hints`. This is an example and likely not part of the spec. How would we handle this? 
+
+# Connectivity
+
+Connectivity describes relationships between lists of atoms and associated metadata: typically, this is between a pair of atoms (a bond). Connectivity is supported across periodic boundaries.
+
+Each connection is defined as:
+
+* `indices`, an ordered list of length `N` of atomic indexes defining the connection
+* `images`, (optional, only relevant if a unit cell lattice is defined) an ordered list of length `3N` describing the periodic image of the connected atom -- by default, this is assumed to be (0,0,0), meaning that the connected atom is in the origin image, and first index is set to (0,0,0) by convention
+* a dictionary of metadata, where each value is a list of length `N` (for example, a key could be "bond_type" and values could be "single", "double", etc.)
+
+For metadata, there are a few pre-defined keys with strict meanings:
+
+* `bond_type` can take values `single`, `double`, `triple` [to replace -- this is just an example], only suitable for connections involving pairs of atoms
```

Comment:
Angle and torsions would need to be handled in a similar way

bond_type defined as is may not be necessary. Single, double, triple ... bonds are defined through the set of parameters associated with each atom type